.gitignore

```diff
@@ -284,3 +284,6 @@ src/startupscriptgenerator/src/**/__debug_bin
 src/startupscriptgenerator/src/**/run.sh
 !**/dynamic/Dockerfile
 !**/dynamic/buster.Dockerfile
+/images/runtime/python/3.10/bullseye.Dockerfile
```

Comment:
There are two lines above in this file that have the following:

```
images/runtime/python/*/buster.Dockerfile
images/runtime/python/*/stretch.Dockerfile
```

Let's delete the lines you've added here and add the following line underneath these two:

```
images/runtime/python/*/bullseye.Dockerfile
```